.github/workflows/publish.yml

```diff
@@ -368,6 +368,76 @@ jobs:
           build-args: |
             IMAGE_VERSION=${{ steps.prep.outputs.version }}
 
+  docker-debian-jdk:
+    needs: [npm, cubestore, native_linux]
+    name: Debian with jdk docker image
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Repo metadata
+        id: repo
+        uses: actions/github-script@v3
+        with:
+          script: |
+            const { data } = await github.repos.get(context.repo)
+            const reg = new RegExp('📊 ', 'ug');
+            data.description = data.description.replace(reg, "")
+            return data
+      - name: Prepare
+        id: prep
+        run: |
+          DOCKER_IMAGE=cubejs/cube
+          VERSION=noop
+          if [ "${{ github.event_name }}" = "schedule" ]; then
+            VERSION=nightly
+          elif [[ $GITHUB_REF == refs/tags/* ]]; then
+            VERSION=${GITHUB_REF#refs/tags/}
+          fi
+          TAGS="${DOCKER_IMAGE}:${VERSION}-debian-jdk"
```

Comment:
Could you please remove "debian" from the label name?

Reply:
done! thank you =)